jobs/rocoto/vrfy.sh

```diff
@@ -88,10 +80,16 @@ if [ $VRFYFITS = "YES" -a $CDUMP = $CDFNL -a $CDATE != $SDATE ]; then
     [[ ! -d $TMPDIR ]] && mkdir -p $TMPDIR
 
     xdate=$($NDATE -${VBACKUP_FITS} $CDATE)
-
-
     export RUN_ENVIR_SAVE=$RUN_ENVIR
     export RUN_ENVIR=$OUTPUT_FILE
+    export COMDAY=$PBS_O_WORKDIR
```

Comment:
$PBS_O_WORKDIR is a `PBS` specific variable and not defined on R&D machines w/ Slurm.
Please verify its compatibility on R&D machine.

Reply:
Should this not be $DATA?

Reply:
Yes. The $PBS_O_WORKDIR is for PBS. Only pre implementation parallel are running the EMC verification jobs. I will revert this change.